product_docs/docs/mongo_data_adapter/5/06_features_of_mongo_fdw.mdx

```diff
@@ -96,3 +96,61 @@ Steps for retrieving the document:
     { "_id" : { "$oid" : "58a1ebbaf543ec0b9054585a" }, "warehouse_id" : 2, "warehouse_name" : "Laptop", "warehouse_created" : { "$date" : 1447229590000 } }
     (2 rows)
     ```
+
+### Accessing nested fields
+
+MongoDB Foreign Data Wrapper allows you to access individual fields within nested JSON documents by mapping the nested structure to columns in a foreign table. 
+This works by mapping the nested structure of the MongoDB document to relational columns in the foreign table definition, using dot notation (key2.subkey21) to reference nested fields.
+You can retrieve these fields from a collection as shown in the following example:
+
+#### Example
+
+```text
+db1> db.test_sub_json.find()
+[
+  {
+    _id: ObjectId('658040214890799d6e0173d0'),
+    key1: 'hello',
+    key2: {
+      subkey21: 'hello-sub1',
+      subkey22: 'hello-sub2',
+      subtstmp: ISODate('2022-12-16T19:16:17.801Z')
+    }
+  }
+]
+```
+
+Steps for retrieving sub-fields from the document:
+
+1.  Create a foreign table. To access a sub-field use the dot (".") in the columnn name as shown below:
+
+```sql
```

Comment:
A suggestion: Whenever I have code snippets in a numbered list, I nest the code snippet within the respective step by indenting the code lines by four spaces. This way, you don't need to enumerate each step in the numbered list. You can mark all steps as 1. but Markdown automatically generates the appropriate number.  The benefit is that if you ever have to add more steps to the instructions, you won't have to worry about fixing all other numbers.

Reply:
Thanks Gabby, yeah I used to do that, but sometimes it would render through as 1. 1. 1. and so on—depending on how I indented snippets in between, but I will double-check if it is necessary on this one.

Reply:
Cool, it worked on this one. 👍